iroh-sync/src/store/fs.rs

```diff
@@ -148,11 +112,15 @@ impl Store {
             let _table = write_tx.open_table(AUTHORS_TABLE)?;
             let mut latest_table = write_tx.open_table(LATEST_TABLE)?;
             let _table = write_tx.open_multimap_table(NAMESPACE_PEERS_TABLE)?;
-            let _table = write_tx.open_table(RECORDS_BY_KEY_TABLE)?;
+            let mut by_key_table = write_tx.open_table(RECORDS_BY_KEY_TABLE)?;
 
             // migration 001: populate latest table if it was empty before
             if latest_table.is_empty()? && !records_table.is_empty()? {
-                migration_001_populate_latest_table(&records_table, &mut latest_table)?;
+                migrations::migration_001_populate_latest_table(&records_table, &mut latest_table)?;
+            }
```

Comment:
we should be committing the transactions between individual migrations

Reply:
Fixed in latest commit